app/javascript/components/pame/PameModal.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <div 
+    :class="['modal-wrapper', { 'modal--active' : isActive }]"
+    @click.self="closeModal()"
+  >
+    <div 
+      class="modal-overlay" 
+    />
```

Comment:
does your formatter add these line breaks? Linter allows single line if only one attribute. No need to fix, just flagging